autofs-5.1.3 - fix mount.nfs blocks on first mount

From: Ian Kent <raven@themaw.net>

The first time an NFS mount is tried mount.nfs tries to start rpc.statd
using systemctl. If /usr/local is being used to provide automounts in
that directory the first mount will hang becuase some directories under
/usr/local are included in the path used by systemctl.

Add rpc-statd.service (and for good measure rpcbind.service) to the
autofs Wants line in the unit file to work around this.

Signed-off-by: Ian Kent <raven@themaw.net>
---
 CHANGELOG                 |    1 +
 samples/autofs.service.in |    2 +-
 2 files changed, 2 insertions(+), 1 deletion(-)

--- autofs-5.0.7.orig/CHANGELOG
+++ autofs-5.0.7/CHANGELOG
@@ -252,6 +252,7 @@
 - add congigure option for limiting getgrgid_r() stack usage.
 - fix unset tsd group name handling.
 - set systemd KillMode to process.
+- fix mount.nfs blocks on first mount.
 
 25/07/2012 autofs-5.0.7
 =======================
--- autofs-5.0.7.orig/samples/autofs.service.in
+++ autofs-5.0.7/samples/autofs.service.in
@@ -1,7 +1,7 @@
 [Unit]
 Description=Automounts filesystems on demand
 After=network.target ypbind.service sssd.service network-online.target remote-fs.target
-Wants=network-online.target
+Wants=network-online.target rpc-statd.service rpcbind.service
 
 [Service]
 Type=forking
